tckdb/backend/app/tests/models/test_bot.py

```diff
@@ -7,19 +7,25 @@
 
 def test_bot_model():
     """Test creating an instance of Bot"""
-    bot_1 = Bot(name='ARC',
-                version='1.1.0',
-                url='https://github.com/ReactionMechanismGenerator/ARC',
-                git_commit='7ba4d74c73198c76c70742de8c254e075200a582',
-                git_branch='master',
-                )
-    assert bot_1.name == 'ARC'
-    assert bot_1.version == '1.1.0'
-    assert bot_1.url == 'https://github.com/ReactionMechanismGenerator/ARC'
-    assert bot_1.git_commit == '7ba4d74c73198c76c70742de8c254e075200a582'
-    assert bot_1.git_branch == 'master'
-    assert str(bot_1) == "<Bot(name='ARC', version='1.1.0', url='https://github.com/ReactionMechanismGenerator/ARC')>"
-    assert repr(bot_1) == "<Bot(id=None, name='ARC', version='1.1.0', " \
-                          "url='https://github.com/ReactionMechanismGenerator/ARC', " \
-                          "git_commit='7ba4d74c73198c76c70742de8c254e075200a582', " \
-                          "git_branch='master')>"
+    bot_1 = Bot(
+        name="ARC",
+        version="1.1.0",
```

Comment:
hard coded data should move to a constant and assertion should be done against him. all this conventions will help us prevent un-stability in the tests in the future. when we have hard coded data like this in the test we will have to look for all the places it is written when we change something, and it will make it way easier to modify the data for the tests.